docs/quick-start.md

```diff
@@ -45,66 +45,159 @@ Follow the steps below to get up and running for the intended configuration elem
+   2. Click Add and give the feature a name. Typically this is based on the configuration snippet or section. E.g. "aaa".
 
 5. Create a Compliance Rule.
 
-    1. Navigate to `Plugins -> Compliance Rules`.
-    2. Click Add and populate the fields, make sure the rule is linked to the feature created previously. See [Configuration Compliance Settings](./navigating-compliance.md#configuration-compliance-settings) for details.
+   1. Navigate to `Plugins -> Compliance Rules`.
+   2. Click Add and populate the fields, make sure the rule is linked to the feature created previously. See [Configuration Compliance Settings](./navigating-compliance.md#configuration-compliance-settings) for details.
 
 6. Execute Compliance Check.
 
-    1. Navigate to `Plugins -> Configuration Compliance`.
-    2. Click on the `Execute` button and select `Compliance`.
-    3. Select what to run the compliance on.
-    4. Run the Job.
+   1. Navigate to `Plugins -> Configuration Compliance`.
+   2. Click on the `Execute` button and select `Compliance`.
+   3. Select what to run the compliance on.
+   4. Run the Job.
 
 > For in-depth details see [Navigating Compliance](./navigating-compliance.md)
+
+# Load Properties from Git
+
+Golden Config properties include: Compliance Features, Compliance Rules, Config Removals, and Config Replacements. They can be created via the UI, API, or alternatively you can load these properties from a Git repository, defined in YAML files following the this directory structure (you can skip any of them if not apply):
+
+```
+├── golden_config
+│   ├── compliance_features
+│   ├── compliance_rules
+│   ├── config_removes
+│   ├── config_replaces
+```
+
+The files within these folders can follow any naming pattern or nested folder structure, all of them will be recursively taken into account. So it's up to you to decide how to you prefer to organize these files (within the previously stated directory structure):
+
+````
+├── golden_config
+│   ├── compliance_features
+│   │   └── all.yml
+│   ├── compliance_rules
+│   │   ├── my_rule_for_cisco_ios
+│   │   │   ├── some_rules.yml
+│   │   │   └── some_other_rules.yml
+│   │   └── juniper_junos.yml
+│   ├── config_removes
+│   │   ├── cisco_ios.yml
+│   │   └── juniper_junos.yml
+│   ├── config_replaces
+│   │   ├── cisco_ios.yml
+│   │   └── juniper_junos.yml
+``
+
+The `YAML` files will contain all the attributes necessary to identify an object (for instance, a `ComplianceRule` is identified by the `feature_slug` and the `platform_slug` together) and the other attributes (the ones that are not used to identify the object). For example:
+
+`compliance_features` example:
+
+```yaml
+---
+- name: "aaa"
+  slug: "aaa"
+  description: "aaa feature"
+````
+
+`compliance_rules` example:
+
+```yaml
+---
+- feature_slug: "aaa"
+  platform_slug: "cisco_ios"
+  config_ordered: true
+  match_config: |
+    aaa
+    line
+    username
+    role
+    tacacs
+  config_type: "CLI"
```

Comment:
What about type "JSON"?  Would this still work?

Reply:
I have not tested, but it should work, there is no filtering